notebooks/aiida_datatypes_viewers.ipynb

```diff
@@ -7,6 +7,17 @@
     "# Displaying AiiDA data types"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from aiida import load_profile\n",
```

Comment:
Can you please revert this change, so that we test that the default profile is loaded in __init__ still?
Also, you could then extend the `test_aiida_datatypes_viewers` test in `tests_notebooks/test_notebooks.py` to check that the deprecation warning is displayed.

Reply:
Good point, I'll try to add a test.